src/scenic/domains/driving/model.scenic

```diff
@@ -17,6 +17,12 @@ If you are writing a generic scenario that supports multiple maps, you may leave
 ``map`` parameter undefined; then running the scenario will produce an error unless the
 user uses the :option:`--param` command-line option to specify the map.
 
+The ``use2DMap`` global parameter determines whether or not maps are generated in 2D. Currently
+3D maps are not supported, but are under development. If 2D mode is enabled, this parameter is
+set to `True`, and otherwise it is set to `False` by default. If 2D mode is disabled, this 
+flag can still be manually set to `True`, allowing full Scenic 3D functionality, albeit with
+2D maps.
```

Comment:
Something like this might be clearer:

```
By default, this parameter is `False` (so that future versions of Scenic will automatically use
3D maps), unless :ref:`2D compatibility mode` is enabled, in which case the default is `True`.
The parameter can be manually set to `True` to ensure 2D maps are used even if the
scenario is not compiled in 2D compatibility mode.
```